app.js

```diff
@@ -0,0 +1,199 @@
+// this is the game board
+const gameBoard = () => {
+
+    const _board = [0,1,2,3,4,5,6,7,8];
+    const winningCombos = [
+      [0, 1, 2],
+      [3, 4, 5],
+      [6, 7, 8],
+      [0, 3, 6],
+      [1, 4, 7],
+      [2, 5, 8],
+      [2, 4, 6],
+      [0, 4, 8]
+    ];
+
+    const setMark = (mark,index) => {
```

Comment:
Let's further simplify this. For instance if in the `if` statement we have a `return` keyword, then definitely we don't need an `else` statement anymore. If the conditions inside the `if` statement are met then surely it will return. What we could have used is either a `ternary` operator or remove the `else` altogether:

```javascript
if(typeof _board[index] === "number") {
 board[index] === mark;
 return true
}
return false;
```